<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-9MJU3PDQ" failures="2" tests="2" name="Test" time="13.266" errors="0" timestamp="2022-01-07T19:12:08 IST">
  <testcase classname="Test.PracticeProject.UsingAssertion" name="validation" time="0.049">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [FEATURED COURSE] but found [FEATURED COURSES]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [FEATURED COURSE] but found [FEATURED COURSES]
at Test.PracticeProject.UsingAssertion.validation(UsingAssertion.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validation -->
  <testcase classname="Test.PracticeProject.UsingAssertion" name="validation" time="0.079">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at Resources.Base.getScreenshot(Base.java:40)
at Test.PracticeProject.Listeners.onTestFailure(Listeners.java:44)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- validation -->
</testsuite> <!-- Test -->
